strict-base-types/strict-base-types.cabal

```diff
@@ -51,7 +51,7 @@ library
       aeson                 >=2.1.2.1 && <2.3
     , base                  >=4.9     && <5
     , quickcheck-instances  >=0.3.24  && <0.4
-    , strict                >=0.5     && <0.5.1
+    , strict                >=0.5     && <0.6
```

Comment:
This is wrong too.

Easier for everyone if you don't touch versions at all.

Reply:
revertion of singular commit pushed.